beacon-chain/rpc/apimiddleware/endpoint_factory.go

```diff
@@ -128,8 +127,6 @@ func (_ *BeaconEndpointFactory) Create(path string) (*apimiddleware.Endpoint, er
 			OnPreSerializeMiddlewareResponseIntoJson: serializeV2Block,
 		}
 		endpoint.CustomHandlers = []apimiddleware.CustomHandler{handleGetBeaconBlockSSZV2}
-	case "/eth/v1/beacon/blocks/{block_id}/root":
-		endpoint.GetResponse = &BlockRootResponseJson{}
```

Comment:
you need to remove the struct as well

Reply:
you don't _need_ to, but yes it would be a good idea

Reply:
If you mean `BlockRootResponseJson` then it's being used by `getSyncMessageBlockRoot`